agent/requestor/src/main.rs

```diff
@@ -8,14 +8,15 @@ use std::sync::{Arc, Mutex};
 use structopt::{clap, StructOpt};
 
 use std::convert::TryFrom;
+use ya_client::model::payment::Account;
 use ya_client::{cli::ApiOpts, cli::RequestorApi, Error};
 
 mod activity;
 mod market;
 mod payment;
 
 const DEFAULT_NODE_NAME: &str = "test1";
-const DEFAULT_TASK_PACKAGE: &str = "hash://sha3:38D951E2BD2408D95D8D5E5068A69C60C8238FA45DB8BC841DC0BD50:http://3.249.139.167:8000/rust-wasi-tutorial.zip";
+const DEFAULT_TASK_PACKAGE: &str = "hash://sha3:d5e31b2eed628572a5898bf8c34447644bfc4b5130cfc1e4f10aeaa1:http://3.249.139.167:8000/rust-wasi-tutorial.zip";
```

Comment:
maybe this fix should be in a separate PR to master so its easier to trace?

Reply:
Separate PR seems too much trouble but I'll make it a separate commit.

Reply:
👍 

great balance :)